Adds docs for ExportMemberGroups Deploy Setting in 3.4.0+ (#2732)
umbraco/UmbracoDocs: Umbraco-Cloud/Deployment/Deploy-Settings/index.md

:::note
This only needs to be done if your Cloud project was created **before** June 18th 2019. Make sure to [follow the instructions](../Umbraco-Forms-on-Cloud/#did-you-create-your-cloud-project-before-june-18th-2019) to succesfully change this setting on all environments.
:::

> ## ExportMemberGroups
> This is available in Deploy 3.4.0+ version and newer. This setting is to be enabled only if you are using an external membership provider for your members and thus it does not make sense to export member groups which would no longer be managed by Umbraco but by an external membership provider.

Reviewer: ```suggestion
## ExportMemberGroups

This is available in Deploy 3.4.0+ version and newer. 

This setting is to be enabled only if you are using an external membership provider for your members. You will not want to export member groups that would no longer be managed by Umbraco but by an external membership provider.
```

> Setting the `exportMemberGroups` to false will no longer export member groups to disk UDA entities, by default if this setting is not present its value will automatically be set to true as most sites use Umbraco's built in membership provider and thus will want the membership groups exported.

Reviewer: ```suggestion
Setting the `exportMemberGroups` to false will no longer export member groups to disk UDA entities. By default if this setting is not present its value will automatically be set to true as most sites use Umbraco's built-in membership provider and thus will want the membership groups exported.
```

:::note
This only needs to be done if your Cloud project was created **before** June 18th 2019. Make sure to [follow the instructions](../Umbraco-Forms-on-Cloud/#did-you-create-your-cloud-project-before-june-18th-2019) to succesfully change this setting on all environments.
:::

## ExportMemberGroups

This is available in Deploy 3.4.0+ version and newer. 

> This setting is to be enabled only if you are using an external membership provider for your members. You will not want to export member groups that would no longer be managed by Umbraco but by an external membership provider.

Author: ```suggestion
This setting is to be defined and set to false only if you are using an external membership provider for your members. You will not want to export member groups that would no longer be managed by Umbraco but by an external membership provider.
```

Good spot, what I was trying to say was more like if this setting exists or is present, but I can understand the confusion.

Let me know what you think of this @sofietoft

Reviewer: Super! Makes sense 💪 Thanks for updating it!
